### DISCUSSION TITLE
How to open tabs in background?

### DISCUSSION THREAD

**Discussion by ghost** [sienori/Tab-Session-Manager#838] [Q&A]
How do you open tabs in the background when you singleclick to open a new page from any session? I don't want to focus on the newly opened tab everytime i click on them.

edit: Now i know you can do it with the middle mouse button but i want to change that function to the normal open behaviour not having to use the mouse wheel.

**Comment by sienori** [comment]
Middle-click a tab to open it in the background.
The option to open tabs in the background by default will be added if there are many requests.
